cmd/check/main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 
 	manifestRequest, err := http.NewRequest("GET", manifestURL, nil)
 	fatalIf("failed to build manifest request", err)
-	manifestRequest.Header.Add("Accept", "application/vnd.docker.distribution.manifest.v2+json")
+	manifestRequest.Header.Add("Accept", "application/vnd.docker.distribution.manifest.v2+json,application/json")
```

Comment:
Based on the findings in issue #40 I suspect it should -- that would map the behavior that Docker seems to be doing. I'll update.